src/devices/nice_flor_s.c

```diff
@@ -0,0 +1,83 @@
+/** @file
+    Nice Flor-s
```

Comment:
`Nice Flor-s remote for car gates.` perhaps? not the dot, also below.

Reply:
"for gates" (people use it for portal gates also). Also added in README.